query/src/servers/http/v1/query/execute_state.rs

```diff
@@ -157,6 +157,13 @@ impl ExecuteState {
         };
         context.attach_query_str(sql);
 
+        // Auth.
```

Comment:
@BohuTANG you can optionally rebase on [it](https://github.com/datafuselabs/databend/pull/3350) if it is ok to merge fast and first.